sdk/core/azure-core-xml/inc/azure/core/xml/xml_wrapper.hpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+namespace Azure { namespace Core { namespace _internal { namespace Xml {
```

Comment:
This file does only have definitions in `_internal` namespace, therefore it should be in `inc/azure/core/xml/internal/xml_wrapper.hpp`